resources/tap.brs

```diff
@@ -61,7 +61,7 @@ end sub
 ' Prints the provided data as a TAP-compliant diagnostic (i.e. prefixed with `# `)
 ' @param {dynamic} arg - the value to print
 sub __tap_diagnostic(arg as dynamic)
-    print m.getInent() "# " arg
+    print m.getIdnent() "# " arg
```

Comment:
This looks like good progress.

Reply:
Dangit, I thought I force-pushed yesterday!

Fixed now.  Nothing to see here, move along pls